.gitignore

```diff
@@ -14,3 +13,5 @@ Vagrantfile
 .DS_Store
 Gemfile.local
 .idea/
+Gemfile.lock
+Gemfile18.lock
```

Comment:
Travis can run builds with provided Gemfile, no need to make some more magic in commands. We have Gemfile18 that contains dependencies for Ruby 1.8 so there is no need for Gemfile.lock